dash_chat_app/lib/features/login/login_screen.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:flutter/material.dart';
+
+// ============= constants ===========
```

Comment:
Documentation comment starts with ///
So if you want to add documentation comment simple
`/// Constants `should be enough.
Please adjust all the other comments accordingly

Reply:
resolved, thank you.